frontend/src/app/forms/forms-routing.module.ts

```diff
@@ -19,6 +19,14 @@ const routes: Routes = [
         (m) => m.ConfirmationPageModule
       ),
   },
+  // Important: Do not route to the Image Viewer on your own. Instead, please use the provided Service for that.
+  {
+    path: 'image-viewer',
```

Comment:
So by manually routing to it, it doesn't work like you mentioned. Do we need an AuthGuard for this then too?

Reply:
Hmmm... Tbh, I think it is fine if we leave it like it is for now since we will only deploy this as a "real" app (here, nobody is able to manually route to a random page). But we should keep this in mind and maybe think of some better solution later on. :)